sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp

```diff
@@ -33,6 +33,16 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
     int m_testPollingTimeOutMinutes = 20;
     std::chrono::milliseconds m_testPollingIntervalMs = std::chrono::minutes(1);
 
+    void CheckSkipHSMForLive()
```

Comment:
Abbreviations longer than 2 letters go PascalCase
```suggestion
    void CheckSkipHsmForLive()
```

Reply:
done